npc/re/quests/quests_dungeons_200.txt

```diff
@@ -4,8 +4,630 @@
+			mes "It won't be easy to find... any traces, so please come to just one.";
+			mes "Then, I would like to thank you.";
+			close;
+		case 2:
+			mes "[Ansuriya]";
+			mes "Did you find any traces?";
+			mes "You've come to the right place. This note is in Werner's handwriting.";
+			next;
+			mes "[Ansuriya]";
+			mes "Thank you. We will hand this over to the secret wing agents for investigation.";
+			mes "Then you should be rewarded for your hard work.";
+			next;
+			mes "[Ansuriya]";
+			mes "Thank you for coming back tomorrow and working with us.";
+			mes "Because you can't just leave this to anyone.";
+			erasequest 16535;
+			erasequest 16545;
+			setquest 16534;
+			getexp 17000000,12000000;
+			close;
+		}
+		end;
+	case 4:
+		mes "[Ansuriya]";
+		mes "Hmm? Have you changed your mind?";
+		next;
+		mes "[Ansuriya]";
+		mes "Well, then! we have to re-organize! Ugh.";
+		mes "Hero, why don't you reconsider calmly?";
+		close;
+	}
+	close;
+
+OnInit:
+	questinfo( QTYPE_QUEST2, QMARK_YELLOW, "checkquest(16531,HUNTING) == 2" );
+	questinfo( QTYPE_QUEST2, QMARK_YELLOW, "checkquest(16533,HUNTING) == 2" );
+	questinfo( QTYPE_QUEST2, QMARK_YELLOW, "isbegin_quest(16535) == 2" );
+
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "checkquest(16530,PLAYTIME) == 2" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "checkquest(16532,PLAYTIME) == 2" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "checkquest(16534,PLAYTIME) == 2" );
+
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "BaseLevel >= 215 && (checkquest(16531,HUNTING) == -1 && checkquest(16530,PLAYTIME) == -1)" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "BaseLevel >= 215 && (checkquest(16533,HUNTING) == -1 && checkquest(16532,PLAYTIME) == -1)" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "BaseLevel >= 215 && (isbegin_quest(16535) == 0 && checkquest(16534,PLAYTIME) == -1)" );
+	end;
+}
+
+amicitia1,57,242,3	script	Something#ami1	4_GC109,{
+	if (isbegin_quest(16535) == 1) {
+		mes "[Something]";
+		mes "There is something on the floor. Shall we take a closer look?";
+		next;
+		mes "[Something]";
+		mes "Research memo in neat handwriting. Werner's handwriting is exactly like this.";
+		next;
+		mes "[Something]";
+		mes "Let's take it and check it out.";
+		setquest 16545;
+		completequest 16535;
+		close2;
+		initnpctimer;
+		disablenpc();
+		end;
+	}
+	mes "[Something]";
+	mes "Something is off the floor.";
+	mes "!it looks like it's just garbage.";
+	close;
+
+OnTimer300000:	// unknown timer
+	stopnpctimer;
+	enablenpc();
+	end;
+}
+amicitia1,152,159,3	duplicate(Something#ami1)	Something#ami2	4_GC109
+amicitia1,262,139,3	duplicate(Something#ami1)	Something#ami3	4_GC109
+amicitia1,36,106,3	duplicate(Something#ami1)	Something#ami4	4_GC109
+amicitia1,244,49,3	duplicate(Something#ami1)	Something#ami5	4_GC109
+
+// amicitia2
+pub_cat,89,32,3	script	Gerbena#ami	4_M_REBELLION2,{
+	if (BaseLevel < 230) {
+		mes "[Gerbena]";
+		mes "Wow, " + strcharinfo(0) + "! Can you sign?";
```

Comment:
Same comment as above:
"Can you sign" sounds like "Can I get an autograph?" or maybe some "object" (which shall be signed) is missing in the sentence.